deps/uv/uv.gyp

```diff
@@ -84,12 +64,6 @@
         'src/version.c'
       ],
       'conditions': [
-        [ 'gcc_version<=44', {
-          # GCC versions <= 4.4 do not handle the aliasing in the queue
-          # implementation, so disable aliasing on these platforms
-          # to avoid subtle bugs
-          'cflags': [ '-fno-strict-aliasing' ],
```

Comment:
The `-fno-strict-aliasing` flag was added to fix compilation warnings when building Node.js with GCC <= 4.4. See https://github.com/joyent/node/pull/9098 for more information. Although I don't like to float this patch, I think we may want to keep it for now and discuss how we could not have to float it (either by upstreaming, or changing the queue implementation).